content/post/personas.md

```diff
@@ -0,0 +1,49 @@
+
+<!--more-->
+
+Defining the typical user of SONAR is part of the element *scope* (user needs). To this aim, we adopted a method based on personas.
+
+### What is a persona?
+
+The *persona* is a well-known concept in the interaction design community. It was introduced 1999 by Alan Cooper[^2], who defines it as follows:
+
+> We make up pretend users and design for them. We call these pretend users personas and they are the necessary foundation of good interaction design. Personas are not real people, but they represent them throughout the design process. They are hypothetical archetypes of actual users. Although they are imaginary, they are defined with significant rigor and  precision. Actually, we don't so much "make up" our personas as discover them as a byproduct of the investigation process. We do, however, make up their names and personal details.
+
+### The 3 main SONAR personas
+
+Each persona is presented below with an image, a name, some minimal context information and the tasks he/she performs on SONAR. The tasks are listed for both services: the shared and the dedicated portals (as described in the [Service](/service) section).
+
+|   |   |
+| ------ | ----------- |
+| <img class="image" src="/images/persona1.svg" alt="Population persona" title="Population persona" style="vertical-align:top;"><br/>**Fabrice**<br/>*Student (University of Kinshasa)* | **Shared or dedicated portal**<ul><li>access full text of publications for free so that I can keep up to date with scientific news</li><li>access publication metadata so that I can include references in my own publications</li><li>search publications using various criteria (author, project, institution, document type, etc.)</li></ul> |
+| <img class="image" src="/images/persona2.svg" alt="Researcher persona" title="Researcher persona" style="vertical-align:top;"><br/>**Gordana**<br/>*Post-doctoral researcher (University of Fribourg - with dedicated SONAR portal)* | **Shared portal**<ul><li>display publicly a list of all my Swiss OA publications so that I can show my scientific accomplishments</li><li>...and all tasks of Fabrice</li></ul>**Dedicated portal**<ul><li>display publicly a list of all my publications (OA or not) related to my institution so that I can show my scientific accomplishments</li><li>upload full-texts of my pre-prints and post-prints so that I comply with the OA policy of my institution</li><li>enter and edit metadata about my publications so that they are searchable</li><li>export a list of all my publications (OA or not) related to my institution or to a specific project so that I can reuse the data</li><li>link my profile to standard identifiers (ORCID) so that my publications metadata is more accurate</li><li>import metadata from other platforms (ORCID, Google Scholar, etc.) so that I don't have to enter it manually</li><li>edit my personal profile</li><li>...and all tasks of Fabrice</li></ul> |
```

Comment:
display publicly -> publicly display (2x)

my scientific accomplishments -> my scholarly record (2x)

... and all tasks of Fabrice -> ... plus the same tasks as Fabrice

my publications metadata -> my publications' metadata